Source/BSN.Commons.AutoMapper/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AutoMapper;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace BSN.Commons.AutoMapper.Extensions
```

Comment:
Please add some test project to ensure this behavior work correctly with downstream AutoMapper profiles.